### DISCUSSION TITLE
UVDESK in windows XAMPP not receiving incoming emails.

### DISCUSSION THREAD

**Discussion by marvelindxb** [uvdesk/community-skeleton#690] [General]
Dear Folks,
I installed a UVdesk in windows XAMPP for testing. I'm not receiving any incoming email, but SMTP is working fine. any fix?

**Comment by komal-sh-27** [comment]
@marvelindxb 

Let us know, you have followed the same blog for uvdesk setup:
https://www.uvdesk.com/en/blog/setup-gmail-mailbox-uvdesk-open-source-helpdesk/

Can you please share your screenshot of the error when you are create any ticket or create an agent on dashboard and also share a screenshots of your configuration setup. Also, let us know which uvdesk version your are using.

Thanks and Regards
Uvdesk Team

**Reply by marvelindxb** [reply]
![image](https://github.com/uvdesk/community-skeleton/assets/142803880/8b2d5fbe-e441-487b-83c8-d8ab80a1f355)
![image](https://github.com/uvdesk/community-skeleton/assets/142803880/56e70595-5911-4c38-82b9-a203ac2def3c)
I have attached the images I configured, followed all the steps except the CRON which is not an extension in XAMPP.

**Reply by marvelindxb** [reply]
Please find the version details attached below.

![image](https://github.com/uvdesk/community-skeleton/assets/142803880/c0499c62-0cf6-43c6-a187-8612404008e7)

**Reply by marvelindxb** [reply]
I suspect the problem is the syncing which is not happening with the main GMAIL inbox. May be the service which is not running can be a reason for the same. I confirmed the extension IMAP is enabled and mail:parse was added as well in php.

**Comment by komal-sh-27** [comment]
@marvelindxb 

> Followed all the steps except the CRON which is not an extension in XAMPP

You will be first refresh mailbox command in your project root directory. Once you make a try with the email on the configured email address in mailbox from another email address. After that refresh the mailbox command in your project root directory using the below command:

`$ php bin/console uvdesk:refresh-mailbox your-mailbox-email-address`

**Comment by komal-sh-27** [comment]
@renatochaves1999 ,

Can you please share your screenshot of the error. When you have created new ticket or reply in the ticket.

**Reply by komal-sh-27** [reply]
Open the **swiftmailer.php** file under **config->packages** in your project.

**Disable** **SSL verification** on the server by using this option and make a try.


```
stream_options:
    ssl:
        verify_peer: false
        verify_peer_name: false
        allow_self_signed: true
```

![image](https://github.com/uvdesk/community-skeleton/assets/63539748/5cbda90d-5776-4528-976e-3c821fe128d1)

After that, you will clear the cache of your project run the below command in your project root directory:

`$ php bin/console c:c`

 
Now you can reply to tickets and check swift mailer is working or not.
